variants/dreamcatcher/variant.h

```diff
@@ -0,0 +1,107 @@
+#undef I2C_SDA
+#undef I2C_SCL
+#define I2C_SDA 16 // I2C pins for this board
+#define I2C_SCL 17
+
+#define I2C_SDA1 45
+#define I2C_SCL1 46
+
+#define LED_PIN 6
+#define LED_STATE_ON 1
+#define BUTTON_PIN 0
+
+#define HAS_TPS65233
+
+// V1 of SubG Switch SMA 0 or F Selector 1
+// #define RF_SW_SUBG1 8
+// V2 of SubG Switch SMA 1 or F Selector 0
+// #define RF_SW_SUBG2 5
+
+#define RESET_OLED 8 // Emulate RF_SW_SUBG1, Use F Connector
+#define VTFT_CTRL 5  // Emulate RF_SW_SUBG2, for SMA swap the pin values
+
+#if OTHERNET_DC_REV == 2206
+#define USE_LR1120
+
+#define SPI_MISO 37
+#define SPI_MOSI 39
+#define SPI_SCK 38
+#define SDCARD_CS 40
+
+#define PIN_BUZZER 48
+
+#define GPS_RX_PIN 10
```

Comment:
Just checking - need HAS_GPS 1?

Reply:
The board does not have a soldered GPS. There's a serial header to connect one. Other uses would be the serial module. This was more a reminder to me what the (changing) pinout for those pins was.

Reply:
i'll change them to comments, so people know what to look for when defining pins for either use case. the header is dead center on the board.